packages/ses-next/styles/globals.css

```
@@ -3,26 +3,182 @@
+  --color-success-content: #05151f;
+  --color-warning: #e0a534;
+  --color-warning-content: #05151f;
+  --color-error: #f0776a;
+  --color-error-content: #1d0705;
+
+  --radius-selector: 0.5rem;
+  --radius-field: 0.6rem;
+  --radius-box: 1rem;
 }
 
 @theme {
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)